lib/msal-node/test/client/ManagedIdentitySources/ServiceFabric.spec.ts

```diff
@@ -78,6 +83,15 @@ describe("Acquires a token successfully via an App Service Managed Identity", ()
             expect(networkManagedIdentityResult.accessToken).toEqual(
                 DEFAULT_USER_SYSTEM_ASSIGNED_MANAGED_IDENTITY_AUTHENTICATION_RESULT.accessToken
             );
+
+            const url: URLSearchParams = new URLSearchParams(
+                sendGetRequestAsyncSpy.mock.lastCall[0]
+            );
+            expect(
+                url.has(
+                    ManagedIdentityUserAssignedIdQueryParameterNames.MANAGED_IDENTITY_CLIENT_ID_2019
```

Comment:
same comment as earlier test, can we check, 

```js
expect(
  url.has(
    ManagedIdentityUserAssignedIdQueryParameterNames.MANAGED_IDENTITY_CLIENT_ID_2017
  )
).toBe(false);

```

Reply:
checking this for all other non ML 2017 sources

Reply:
done